drivers/espi/espi_mchp_xec.c

```diff
@@ -258,6 +258,71 @@ static bool espi_xec_channel_ready(struct device *dev, enum espi_channel ch)
 	return sts;
 }
 
+static int espi_xec_read_lpc_request(struct device *dev,
+				     enum lpc_peripheral_opcode op,
+				     u32_t  *data)
+{
+	ARG_UNUSED(dev);
```

Comment:
Do we need to check peripheral channel status and if not enabled/ready return an error?